indico/modules/events/tracks/templates/_track_list.html

```diff
@@ -0,0 +1,91 @@
+{% from 'message_box.html' import message_box %}
+
+{% macro render_track_list(event) %}
+    {% if not event.tracks %}
+        {% call message_box('info', classes='no-rules-placeholder') %}
+            {% trans %}No tracks have been added yet.{% endtrans%}
+        {% endcall %}
+    {% else %}
+        <ul id="track-list" data-url="{{ url_for('.sort_tracks', event) }}">
+            {% for track in event.tracks|sort(attribute='position') %}
+                <li class="track-row i-box" data-id="{{ track.id }}">
+                    <div class="ui-i-box-sortable-handle"></div>
+                    <div class="i-box-header">
+                        <span class="i-box-title">{{ track.title }}</span>
+                        <div class="i-box-buttons toolbar right">
+                            <div class="group">
+                                <a href="#" class="i-button icon-edit" title="{% trans %}Edit track{% endtrans %}"
+                                   data-href="{{ url_for('tracks.edit_track', track) }}"
+                                   data-title="{% trans title=track.title %}Edit {{ title }}{% endtrans %}"
+                                   data-reload-after
+                                   data-ajax-dialog></a>
+                                <a href="#" class="i-button icon-remove js-delete-item"
+                                   title="{% trans %}Delete track{% endtrans %}"
+                                   data-href="{{ url_for('tracks.delete_track', track) }}"
+                                   data-confirm="{% trans %}Are you sure you want to delete this track?{% endtrans %}"
+                                   data-title="{% trans %}Delete track{% endtrans %}"
+                                   data-method="DELETE"
+                                   data-update="#track-list"></a>
+                            </div>
+                        </div>
+                        <span class="i-box-description">{{ track.code }}</span>
+                    </div>
+                    {% if track.description %}
+                        <div class="i-box-content">
+                            {{ track.description }}
+                        </div>
+                    {% endif %}
+                </li>
+            {% endfor %}
+        </ul>
+    {% endif %}
+
+    <script>
+        var heightLimit = 50;
+        $(document).ready(function() {
```

Comment:
this is most likely not needed if everything you care about is already defined in the DOM